### DISCUSSION TITLE
Anyone knows how to open commands when using the default dashboard

### DISCUSSION THREAD

**Discussion by vitorf7** [LazyVim/LazyVim#1804] [Q&A]
I was trying to do `nvim -c ":Octo pr edit 1234"` and when neovim opens I get:

```shell
OctoEditable   xxx cleared
Press ENTER or type command to continue

Error executing vim.schedule lua callback: ...share/nvim/lazy/octo.nvim/lua/octo/model/octo-buffer.lua:77: Buffer is not 'modifiable'
stack traceback:
        [C]: in function 'nvim_buf_set_lines'
        ...share/nvim/lazy/octo.nvim/lua/octo/model/octo-buffer.lua:77: in function 'clear'
        ...share/nvim/lazy/octo.nvim/lua/octo/model/octo-buffer.lua:99: in function 'render_issue'
        ...iante/.local/share/nvim/lazy/octo.nvim/lua/octo/init.lua:246: in function 'create_buffer'
        ...iante/.local/share/nvim/lazy/octo.nvim/lua/octo/init.lua:82: in function 'cb'
        ...iante/.local/share/nvim/lazy/octo.nvim/lua/octo/init.lua:107: in function 'cb'
        ...te/.local/share/nvim/lazy/octo.nvim/lua/octo/gh/init.lua:119: in function ''
        vim/_editor.lua: in function <vim/_editor.lua:0>
```

If I try again doing `:Octo pr edit 1234` I get:
```shell
   Error  15:57:43 msg_show.lua_error   Octo pr edit 58 Error detected while processing CursorMoved Autocommands for "<buffer=1>":
15:57:43 msg_show   Octo pr edit 58 Error executing lua callback: ...re/nvim/lazy/dashboard-nvim/lua/dashboard/theme/doom.lua:132: Cursor position outside buffer
stack traceback:
	[C]: in function 'nvim_win_set_cursor'
	...re/nvim/lazy/dashboard-nvim/lua/dashboard/theme/doom.lua:132: in function <...re/nvim/lazy/dashboard-nvim/lua/dashboard/theme/doom.lua:122>
   Error  15:57:44 msg_show.lua_error Error executing vim.schedule lua callback: ...share/nvim/lazy/octo.nvim/lua/octo/model/octo-buffer.lua:77: Buffer is not 'modifiable'
stack traceback:
	[C]: in function 'nvim_buf_set_lines'
	...share/nvim/lazy/octo.nvim/lua/octo/model/octo-buffer.lua:77: in function 'clear'
	...share/nvim/lazy/octo.nvim/lua/octo/model/octo-buffer.lua:99: in function 'render_issue'
	...iante/.local/share/nvim/lazy/octo.nvim/lua/octo/init.lua:246: in function 'create_buffer'
	...iante/.local/share/nvim/lazy/octo.nvim/lua/octo/init.lua:82: in function 'cb'
	...iante/.local/share/nvim/lazy/octo.nvim/lua/octo/init.lua:107: in function 'cb'
	...te/.local/share/nvim/lazy/octo.nvim/lua/octo/gh/init.lua:119: in function ''
	vim/_editor.lua: in function <vim/_editor.lua:0>
```

So I decided to try out disabling `dashboard` and use `alpha`.
Now if I run the same command it works correctly.

I have also tried (while using `dashboard` plugin and not alpha) `nvim -c ":Telescope find_files"` as a test and although the pop up opens, it is broken and does not allow you to find anything. Something in the configuration of the default dashboard seems to be making things not modifiable and therefore breaking when you use plugins when opening neovim with a command.

Anyone knows how to get around this or fix it?

**Comment by dpetka2001** [comment]
Have you tried opening neovim with a file after your command? Example: `nvim -c 'your_command' file.lua`. Does it still behave the same?

**Reply by vitorf7** [reply]
I just tried and actually it works, however that really is a bit of a pain to be honest as most of these commands are supposed to be generic and I would not know what files are available.
For example I have a keybind for my `gh-dash` which open a new tmux window and then opens neovim with octo to edit a PR.
```yaml
prs:
  - key: O
    command: >
      tmux new-window -n {{.RepoName}} -c {{.RepoPath}} 'nvim -c ":Octo pr edit {{.PrNumber}}"'
```

**Reply by dpetka2001** [reply]
It seems that it has something to do with the dashboards that are enabled at the time. Not sure if this is a Lazyvim issue or maybe an issue should be opened at the plugin's repo. Better ping the maintainer to share his opinion as I cannot provide any more insight.

**Reply by vitorf7** [reply]
Yeah I believe you are correct. @folke any ideas?

**Reply by Eryx5502** [reply]
I was facing the same issue and the only option I found was to open a new file and delete the buffer afterwards as a workaround:
```yaml
  prs:
    - key: o
      command: >
        tmux new-window -c {{.RepoPath}} 'NVIM_APPNAME=lazyvim nvim +":silent Octo pr edit {{.PrNumber}}" +bd1 temp'
```
